packages/graphql/src/schema/query.ts

```diff
@@ -105,8 +105,30 @@ export const Query = builder.queryType({
       userTags: asUser.prismaField({
         nullable: true,
         type: [Tag],
-        resolve(_, __, ___, ctx) {
-          return QuestionService.getUserTags(ctx)
+        async resolve(_, __, ___, ctx) {
```

Comment:
moved some of the very simple services here. when we split up the long query and mutation files anyways, we might as well merge some of the simple services, so we do not have it distributed across so many files.